src/tasks/LibraryBuilder/LibraryBuilder.cs

```diff
@@ -93,6 +93,14 @@ public bool IsSharedLibrary
     [Output]
     public string OutputPath { get; set; } = ""!;
 
+    /// <summary>
+    /// The set of export symbols that need to be understood when building for a static library
+    ///
+    /// Note, this does not apply when IsSharedLibrary == true.
+    /// </summary>
+    [Output]
+    public string[] ExportedSymbols { get; set; } = Array.Empty<string>();
```

Comment:
Wouldn't it make sense to always output the exported functions? Feels less complicated to not condition this and it might be useful for users of task to get that list when building shared libraries as well.